### ISSUE TITLE
Indicate which file is failing

### ISSUE THREAD

**Issue by MoOx** [DSchau/prettier-markdown#7]
Currently, if a file fail when you use a glob, you don't know which file is responsible for the failure.

```console
❯ prettier-markdown "packages/preset-react-app/**/*.md"
✖ SyntaxError: Unexpected token, expected , (3:13)
  1 | {
  2 |   ...
> 3 |   "phenomic": {
    |             ^
  4 |     "presets": [
  5 |       "@phenomic/preset-react-app"
  6 |     ]
```

It would be nice to have the filename :)

**Comment by DSchau** [comment]
Agreed! PRs welcome and appreciated.

Also seems related (to some extent) to #5

**Comment by DSchau** [comment]
Hi @MoOx 

Just published as `1.2.0-rc.0` so you can install with `npm install -g @dschau/prettier-markdown@1.2.0-rc.0`

I'm testing it locally, but it should be good to go.

If you could, would you want to validate that I've fixed both issues?

**Comment by DSchau** [comment]
The result should look like this (it'll skip failures, and handle any other file it can)

```
Malformed syntax detected in /__fixtures__/file-with-errors.md
  1 | class Hello extends React.Component {
  2 |   render() {
> 3 |     ...
    |     ^
  4 |   }
  5 | }
Prettified 13 files
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/css.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/front-matter.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/graphql.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/javascript-multiple-same.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/javascript-multiple.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/javascript.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/json.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/kitchen-sink.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/less.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/react.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/scss.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/typescript-with-highlighting.md
✔︎ Updated /Users/dschau/Code/Personal/prettier-markdown/__fixtures__/typescript.md
```
